addons/l10n_account_edi_ubl_cii_tests/tests/test_xml_cii_fr.py

```diff
@@ -182,7 +182,7 @@ def test_export_import_invoice(self):
                         <PaymentReference>___ignore___</PaymentReference>
                 </xpath>
             ''',
-            expected_file='from_odoo/facturx_out_invoice.xml',
+            expected_file_path='from_odoo/facturx_out_invoice.xml',
```

Comment:
remove the keywords instead. It's not an optional parameter

Reply:
this one is cherry picked from Brice's commit in the OIOUBL PR. I will re-cherry pick it once he changes it